### DISCUSSION TITLE
Primefaces FREYA Sidebar Static Menu

### DISCUSSION THREAD

**Discussion by JeanMarieK** [primefaces/community#2347] [PrimeFaces Templates]
Hello, can someone please help me enable the static sidebar menu? By default, it is disabled, which requires the user to hover on the sidebar 
menu and click a little radio button to make it static for that session.

![sidebar menu](https://github.com/user-attachments/assets/3ed9bc80-06e3-45f8-a53f-1578367ed1ae)

I want it to appear like this by default
![static sidebar](https://github.com/user-attachments/assets/3f021d6d-ac48-4406-8659-9c65c1fa64fa)

**Comment by melloware** [comment]
@JeanMarieK i don't own Freya but i own other templates.  And typically in the Showcase it has a java class called `GuestPreferences.java` that controls what is the defaults of the theme.

```java
@Named
@SessionScoped
public class GuestPreferences implements Serializable {

    private String layout = "static";

    private String componentTheme = "blue";

    private String menuTheme = "darkgray";

    private String logoColor = "white";

    private String scheme = "light";

    private String inputStyle = "outlined";

    private List<MenuTheme> menuThemes;

    private List<ComponentTheme> componentThemes;
```

Just look where that is used and change `layout` or `mode` or whatever the default variables are.
